### ISSUE TITLE
[v11.0] web_digital_sign canvas width 100% according to user screen [Suggest Improvement]

### ISSUE THREAD

**Issue by Boss1669** [JayVora-SerpentCS/SerpentCS_Contributions-v8#335]
Suggest Improvement
jSignature to have width = 100% so that it will be responsive on mobile as view , and test sign accordingly
**Expected Results are like this Portrait**
![pasted image 0](https://user-images.githubusercontent.com/26357404/55966908-80181f80-5ca3-11e9-903e-bda099a68182.png)
**Expected Results are like this Landscape**
![pasted image 0](https://user-images.githubusercontent.com/26357404/55966927-8c9c7800-5ca3-11e9-8dad-414f44bcdf32.png)

**Comment by JayVora-SerpentCS** [comment]
Thank you for the suggestion.

**Comment by JayVora-SerpentCS** [comment]
cc @nikitavaghela

**Comment by nikitavaghela** [comment]
@Boss1669  in my system issue is not generated can you show steps or video?

**Comment by Boss1669** [comment]
Hi, Here is a link to video. In Our system we have integrate sign into invoice
PC VIEW 
1. You can see that the box of for convas is this long, 
But where the user can sign is only half of the box

MOBILE VIEW
2.If we change to mobile view the box for canvas is not 100% width to the user screen
so therefore user can sign way off from screen

[](https://youtu.be/dbnsTCmwxMY)

**Comment by Boss1669** [comment]
https://youtu.be/dbnsTCmwxMY

**Comment by nikitavaghela** [comment]
Thank you @Boss1669  for video. I will check this issue.

**Comment by nikitavaghela** [comment]
Hello @Boss1669 

I checked this issue and tried to fix this issue by updating canvas width but there is raise some more issue because of canvas width like following.
*  If I update canvas width 100% it affects on the signature line  for example https://prnt.sc/ntr0h9
*  Updated width also affects when edit existing signature like https://prnt.sc/ntr51c.
* If I update canvas width then if I switch the view from mobile to PC and PC to a mobile then the signature is disabled and if open existing signature sometimes it does not display an existing signature.

To fix this issue I tried to set dynamic width like 100% or window size but it's not working I also check with static width but in static width is also raised the issue when switch view.
